scripts/startNetwork/registerUsers/registerOrdererOrgUsers.sh

```diff
@@ -25,7 +25,7 @@ fabric-ca-client register \
   --id.name $ADMIN_ORG0_IDENTITY_USER \
   --id.secret $ADMIN_ORG0_IDENTITY_PASSWORD \
   --id.type admin \
-  --id.attrs "hf.Registrar.Roles=client,hf.Registrar.Attributes=*,hf.Revoker=true,hf.GenCRL=true,admin=true:ecert,abac.init=true:ecert" \
+  --id.attrs "hf.Registrar.Roles=client,hf.Registrar.Attributes=*,hf.Revoker=true,hf.GenCRL=true,admin=true:ecert" \
```

Comment:
```suggestion
  --id.attrs "hf.Registrar.Roles=client,hf.Registrar.Attributes='*',hf.Revoker=true,hf.GenCRL=true,admin=true:ecert" \
```
To keep it consistent